frontend/packages/console-dynamic-plugin-sdk/src/app/k8s/api-discovery/discovery-cache.ts

```diff
@@ -0,0 +1,58 @@
+import * as _ from 'lodash';
+
+const SDK_API_DISCOVERY_RESOURCES_LOCAL_STORAGE_KEY = 'sdk/api-discovery-resources';
+
+export const cacheResources = (resources) =>
```

Comment:
Curious - is there any difference between this function and where we got it from the console aside from the local storage key? https://github.com/openshift/console/blob/master/frontend/public/module/k8s/get-resources.ts#L34

Wondering if we could make that a parameter to the function and de-dupe unless we want to explicitly keep this private in both locations.

Reply:
my intention was to keep this private as it does not make sense to expose it from SDK.